.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Static Site
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build project
```

Comment:
are we going to run unit testing as well in the script or will it be another script? The flow is when someone make PR, it should run build and all tests and it should block the PR if anything goes wrong